admin_resources.md

```diff
@@ -0,0 +1,33 @@
+# Administrative Resources
+
+## Leave (vacation / personal / sick time)
```

Comment:
Maybe just delete this section since we have https://github.com/pulibrary/dls-handbook/blob/main/vacation.md now

Reply:
Or a link to the handbook page?

Reply:
Thanks all, I edited and added the link to the vacation page. @eliotjordan @hackartisan